### PR TITLE
Issue/37 feat create getContent and corresponding Test

### PR DESCRIPTION
**Pull request by ElisabethChen**

### CODE REVIEW

File: ci/src/test/java/EmailTest.java [hunk 1]
```diff
@@ -43,4 +45,31 @@ public void TestEmailFailOneGivenRecipients() {
...
+    @Test
+    public void TestGetContentAllSuccess() {
+        String commitID = "33ds2e12saa";
+        String expected = String.format("Status for commit %s:\n \tCompile: %s\n\tTest: %s ", commitID, "Success", "Success");
+        String actual =email.getContent(new PushStatus(commitID, true, true, "", ""));
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    public void TestGetContentComileSuccessTestFail() {
```
Comment (YaourtNature83) [inline ci/src/test/java/EmailTest.java]: did you mean "ContentCompileSuccess" ?

File: ci/src/test/java/EmailTest.java [hunk 2]
```diff
@@ -43,4 +45,31 @@ public void TestEmailFailOneGivenRecipients() {
...
+        String commitID = "33ds2e12saa";
+        String expected = String.format("Status for commit %s:\n \tCompile: %s\n\tTest: %s ", commitID, "Success", "Fail");
+        expected += "\n\nError message for test:\n\ttest failed";
+        String actual =email.getContent(new PushStatus(commitID, true, false, "", "test failed"));
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    public void TestGetContentComileFailTestFail() {
```
Comment (YaourtNature83) [inline ci/src/test/java/EmailTest.java]: idem

File: ci/src/test/java/EmailTest.java [hunk 3]
```diff
@@ -43,4 +45,31 @@ public void TestEmailFailOneGivenRecipients() {
         email.changeRecipients(recipients);
         assertFalse(email.send("test failed sent email, changed to one recipients"));
     }
+
+    @Test
+    public void TestGetContentAllSuccess() {
+        String commitID = "33ds2e12saa";
```
Comment (YaourtNature83) [inline ci/src/test/java/EmailTest.java:51]: Is the commit ID supposed to be hard coded ?

**Review by YaourtNature83 — CHANGES_REQUESTED** [review]
Overall good I think.

File: ci/src/test/java/EmailTest.java (hunk 1 above)
Comment (ElisabethChen) [reply ci/src/test/java/EmailTest.java]: ups, will fix it

File: ci/src/test/java/EmailTest.java (hunk 3 above)
Comment (ElisabethChen) [reply ci/src/test/java/EmailTest.java:51]: I thought that we get the commitID from the inputs and then set it in the pushStatus. Here I just write some random commitID for test

**Review by cadye — APPROVED** [review]
looks good

**Pull request merged**